content/features/featuresDeepDive/lights/lights_introduction.md

```diff
@@ -78,6 +92,8 @@ In these playgrounds see how the specular color (green) is combined with the dif
 
 <Playground id="#20OAV9#5" title="Hemispheric Light Example" description="Simple Example of adding a Hemispheric Light to your scene." image="/img/playgroundsAndNMEs/divingDeeperLightsIntro4.jpg" isMain={true} category="Lights"/>
 
+<Playground id="#T7FXR8#11" title="Rectangular Area Light Example" description="Simple Example of adding a Rectangular Area Light to your scene." image="/img/playgroundsAndNMEs/areaLightStandardExample.jpg" isMain={true} category="Lights"/>

```

Comment:
The example has a fly camera in it, which is a little hard to use to move the scene around to see how the area lights are illuminating and reflecting in the floor. I think an ArcRotate would be easier to use.